openmetadata-ui/src/main/resources/ui/src/utils/ServiceUtils.tsx

```diff
@@ -874,6 +887,8 @@ export const getCountLabel = (serviceName: ServiceTypes) => {
       return 'Pipelines';
     case ServiceCategory.ML_MODEL_SERVICES:
       return 'Models';
+    case ServiceCategory.OBJECT_STORE_SERVICES:
+      return 'Containers';
```

Comment:
Did we miss localisation in this file?

Reply:
We have localization in this file, but for this method, we don't have it, so I will update it.